src/homepage/components/kaiUiComponents/Header/Header.scss

```diff
@@ -0,0 +1,32 @@
+
+
+// $kai-header-height: 2.8rem;
+
+.kai-heade {
```

Comment:
Can we have proper names, also please separate generic styles from specific styles, and name accordingly